src/suffer.cpp

```diff
@@ -954,65 +1057,42 @@ void suffer::from_sunburn( Character &you )
         affected_part_names.push_back( bp_name );
     }
 
-    std::string all_parts_list = enumerate_as_string( affected_part_names );
-
-    std::string message;
-    if( you.has_trait( trait_ALBINO ) || you.has_effect( effect_datura ) ) {
-        //~ %s is a list of body parts.  The plurality integer is the total
-        //~ number of body parts
-        message = n_gettext( "The sunlight is really irritating your %s.",
-                             "The sunlight is really irritating your %s.",
-                             affected_bodyparts.size() );
-    } else if( you.has_trait( trait_SUNBURN ) ) {
-        //~ %s is a list of body parts.  The plurality integer is the total
-        //~ number of body parts
-        message = n_gettext( "The sunlight burns your %s.",
-                             "The sunlight burns your %s.",
-                             affected_bodyparts.size() );
-    }
-    you.add_msg_if_player( m_bad, message, all_parts_list );
+    const std::string all_parts_list = enumerate_as_string( affected_part_names );
 
-    // Wake up from skin irritation/burning
-    if( you.has_effect( effect_sleep ) ) {
-        you.wake_up();
-    }
+    // s is a list of body parts.  The plurality integer is the total
```

Comment:
Comments starting with ~ are for translators and should go directly before the translatable string, in this case right in front of `ngettext`.

Reply:
I assume in the line above is enough and it does not need to be inside the `warn_and_wake_up` call? If so, this is resolved.